src/redux/configureStore.js

```diff
@@ -0,0 +1,15 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+  // additional reducers could be added here
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(logger),
+);
```

Comment:
- [ ] Good use of redux `logger` here. Kudos 💯

Reply:
Thanks